packages/core/package.json

```diff
@@ -17,6 +17,7 @@
     "test": "svelte-kit sync && vitest run",
     "test:watch": "vitest"
   },
+  "module": "./dist/index.js",
```

Comment:
This seems suspicious, since `module` is a non-standard field and has been supplanted by `exports`. Why was this needed? It seems wrong, because `dist/index.js` (to my knowledge) cannot be imported in a non-Svelte project

Reply:
I think something was breaking in CI but agreed, replaced with the standard `{ exports: { import: "..." } }` field